src/combobox/ComboBox.ts

```diff
@@ -169,19 +162,19 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 	private _onInputKeyDown(event: KeyboardEvent) {
 		const { results = [] } = this.properties;
 
-		switch (event.keyCode) {
-			case keys.up:
+		switch (event.which) {
```

Comment:
`which` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which)